portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> comments;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    comments = new ArrayList<>();
+    comments.add("These");
+    comments.add("are");
+    comments.add("test");
+    comments.add("comments");
+
+    String json = "[";
+    for (String comment : comments)
+      json = json.concat("comment: " + comment + "},");
```

Comment:
Should there be a matching `"{"` after `"comment: "`?

Reply:
I fixed that in the next comment along with a few other formatting mistakes that lead to run time errors.